src/Prima/Pyxis/Form.elm

```diff
@@ -1688,3 +1946,21 @@ canShowError : model -> FormField model msg -> Bool
 canShowError model ((FormField opaqueConfig) as config) =
     (not << isValid model) config
         && ((not << isPristine model) config || hasNotEmptyValidation opaqueConfig)
+
+
+pickWarning : model -> FormFieldConfig model msg -> List String
+pickWarning model opaqueConfig =
+    List.filterMap
+        (\rule ->
+            if validateWarning model opaqueConfig rule then
+                Nothing
+
+            else
+                (Just << Validation.pickError) rule
+        )
+        (pickValidationRules opaqueConfig)
+
+
+canShowWarning : model -> FormField model msg -> Bool
```

Comment:
magari **should**ShowWarning

Reply:
l'ho scritto come canShowError

Reply:
@lucabell allora aggiornerei anche `canShowError` in `shouldShowError`